games-emulation/mame/patches/mame-0.259.patchset

```diff
@@ -0,0 +1,881 @@
+From 38eced877589a5acb70c29b4673a08e19f44c429 Mon Sep 17 00:00:00 2001
+From: Alex Brown <mail@alexbrown.info>
+Date: Fri, 13 Oct 2023 11:46:08 +0100
+Subject: Attempt at porting 0.226 patch to 0.259
```

Comment:
Could you keep credit to the original authors of the pachset here? Add yours where changes were done. TIA

Reply:
Yes, of course! Sorry, forgot about this. I've been using your updated patchset with the correct author and updating that locally.

Reply:
Updated with current fixes. Please do let me know if there's anything else I need to correct.